app/api/dao/mentorship_relation.py

```diff
@@ -1,11 +1,11 @@
-from datetime import datetime, timedelta
+is not Nonefrom datetime import datetime, timedelta
```

Comment:
Are you sure you wanted to make this change?

Reply:
It got pushed by mistake